zend_abstract_interface/zai_string/string.h

```diff
@@ -9,4 +11,9 @@ typedef struct zai_string_view_s {
 #define ZAI_STRL_VIEW(cstr) \
     (zai_string_view) { .len = sizeof(cstr) - 1, .ptr = (cstr) }
 
+#define ZAI_STRING_EMPTY \
+    (zai_string_view) { .len = 0, .ptr = "" }
+
+static inline bool zai_string_stuffed(zai_string_view s) { return s.ptr && s.len; }
```

Comment:
```suggestion
static inline bool zai_string_stuffed(zai_string_view s) { return s.len && s.ptr; }
```

I think it's slightly more cautious to not look at the ptr if `len` is 0.

Reply:
I'm not doing *s.ptr, just checking whether it's non-NULL

Reply:
True.